config/validator/service/validator-proposer-settings.go

```diff
@@ -0,0 +1,35 @@
+package validator_service_config
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	field_params "github.com/prysmaticlabs/prysm/config/fieldparams"
+)
+
+// ValidatorProposerSettingsConfig is the struct representation of the JSON or YAML payload set in the validator through the CLI.
+// ProposeConfig is the map of validator address to fee recipient options all in hex format.
+// DefaultConfig is the default fee recipient address for all validators unless otherwise specified in the propose config.required.
+type ValidatorProposerSettingsConfig struct {
```

Comment:
I find `ValidatorProposer` a bit mouthful and expressive. Why not just `Proposer`? Examples:
- `config/validator/service/proposer-settings.go`
- `ProposerSettingsConfig`
- `ProposerOptionsConfig`

Reply:
sure we can call it just proposer, perhaps I should change the flag names too then